vendor/github.com/AudriusButkevicius/pfilter/misc.go

```diff
@@ -0,0 +1,36 @@
+package pfilter
+
+import (
+	"fmt"
+	"net"
+	"sync"
+)
+
+var (
+	maxPacketSize = 2048
```

Comment:
Random number larger than most MTUs? 
Have a better idea? I guess I could explicitly set MTU to 1500 in the KCP library.